crates/ln-dlc-node/src/ln/event_handler.rs

```diff
@@ -400,7 +400,32 @@ impl EventHandler {
                     expected_outbound_amount_msat,
                     "Intercepted HTLC"
                 );
-                // TODO: check if there is a channel present
+
+                // if we have already a channel with them, we try to forward the payment.
+                // TODO: here we would need to increase the channel size if the channel is too small
+                if let Some(channel) =
```

Comment:
🤔 Isn't the `requested_next_hop_scid` the fake channel id that we have generated with the invoice? In that case, wouldn't that `scid` different for every invoice? If yes, I guess that check should rather check if there already exists a channel with the target_node.